engine/src/test/java/net/jqwik/api/EMailsTests.java

```diff
@@ -0,0 +1,228 @@
+package net.jqwik.api;
+
+import com.tngtech.archunit.thirdparty.com.google.common.net.*;
+
+import static net.jqwik.api.ArbitraryTestHelper.*;
+
+public class EMailsTests {
```

Comment:
Test classes need not be public. All (most) others in the jqwik codebase are not.